README.md

```diff
@@ -10,14 +10,14 @@ them](https://www.facebook.com/business/help/606443329504150?helpref=faq_content
 
 ## Development
 
-Please refer to our [coding standards](docs/coding-standards.md) information for code styles, naming conventions and other methodologies.
+Please refer to our [coding standards](docs/coding-standards.md) for code styles, naming conventions and other methodologies.
 
 ### Requirements
-* python 3.7 (suggest using
-  [virtualenv](https://docs.python-guide.org/dev/virtualenvs/))
-* Postgres - even if you are using sqlite for development, requirements.txt installs
+* python 3.7 (we recommend [virtualenv](https://docs.python-guide.org/dev/virtualenvs/))
+* PostgreSQL - even if you are using sqlite for development, requirements.txt installs
   psycopg2 which [requires libpq](https://www.psycopg.org/docs/install.html#build-prerequisites). The 
   following should work:
+    * [On Windows](https://www.postgresql.org/download/windows/)
```

Comment:
Is the only way to download/install Postgres on Windows by downloading?

Reply:
There are some package managers out there, but I think this is the easiest and a more convenient way